Cargo.toml

```diff
@@ -13,18 +13,18 @@ vendored = ["openssl/vendored"]
 [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies]
 security-framework = "0.3.1"
 security-framework-sys = "0.3.1"
-lazy_static = "1.0"
+lazy_static = "1.4"
 libc = "0.2"
-tempfile = "3.0"
+tempfile = "3.1"
 
 [target.'cfg(target_os = "windows")'.dependencies]
-schannel = "0.1.13"
+schannel = "0.1.16"
 
 [target.'cfg(not(any(target_os = "windows", target_os = "macos", target_os = "ios")))'.dependencies]
-log = "0.4.5"
-openssl = "0.10.15"
-openssl-sys = "0.9.30"
+log = "0.4.8"
+openssl = "0.10.24"
+openssl-sys = "0.9.49"
 openssl-probe = "0.1"
 
 [dev-dependencies]
-hex = "0.3"
+hex = "0.4"
```

Comment:
All of the bumps here other than hex don't do anything different - the new versions are semver-compatible with the old ones.